src/pane.coffee

```diff
@@ -492,8 +492,9 @@ class Pane extends Model
   saveItemAs: (item, nextAction) ->
     return unless item?.saveAs?
 
-    itemPath = item.getPath?()
-    newItemPath = atom.showSaveDialogSync(itemPath)
+    saveOptions = item.getSaveOptions?() or {}
```

Comment:
Usually we use `? {}` instead of `or {}` since it handles fall backs pretty well.